lacework-agent/values.yaml

```diff
@@ -2,7 +2,7 @@
 image:
   registry: docker.io
   repository: lacework/datacollector
-  tag: 5.4.1
+  tag: ""
```

Comment:
@cirego double checking this is intentional?

Reply:
No, that tag should say 5.4.2.

Reply:
Cool, I'll update it. Heads up that I didn't modify the Helm charts at all after they were generated. Does this mean we need to fix our templates?